src/com/opc/opcconnectionimpl.cpp

```diff
@@ -78,12 +88,12 @@ void COpcConnectionImpl::addItemList(const char* paGroupName, std::vector<std::s
   //we assume all the items in pa_lNewItems are of same group
   const char * groupName = nullptr;
   TItemDataList itemList;
-  unsigned int nreadSize = paReadItems.size(); 
+  DEVLOG_INFO("adding items in COpcConnectionImpl[%s]\n", paGroupName);
+  size_t nreadSize = paReadItems.size();
```

Comment:
I know you just updated the line to `size_t`, but could you please remove the leading `n`, as this is a remnant of our old naming scheme

Reply:
Yes. I changed it.